src/main/java/io/druid/data/input/impl/StringInputRowParser.java

```diff
@@ -35,20 +36,35 @@
  */
 public class StringInputRowParser implements ByteBufferInputRowParser
 {
+  private final static Charset defaultCharset = Charsets.UTF_8;
```

Comment:
also, ```private static final``` is somewhat more standard (900+ occurrences vs 11 occurrences for the other way)